Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc

```diff
@@ -510,6 +520,7 @@ void TrackerGeometryCompare::compareSurfaceDeformations(TTree* refTree, TTree* c
       for (int npar = 0; npar < int(inputDpar2.size()); ++npar) {
         if (inputRawid1 == inputRawid2) {
           surfDeform_[npar] = inputDpar2.at(npar) - inputDpar1.at(npar);
+          std::cout << inputDpar2.at(npar) - inputDpar1.at(npar) << std::endl;
```

Comment:
Is this a remnant of some private debug, intended to be removed?

Reply:
Yeah. If tests are fine I would merge though.

Reply:
At some point this line must be removed, then. I would personally do it now, and backport the amended PR.

However, if you think it can speed up the integration (of the 14_0_X backport, which is the relevant one), one could do the following:
- Merge this PR as such
- Prepare the backport PR without the line with the `cout` (*)
- Submit another PR in master which removes this line

(*) even if it is not a verbatim backport, the difference of just one comment line cannot prevent the backport

Reply:
Okay, pushing one more commit with the debug line removed